src/main/java/frc/robot/commands/IntakeCommandAxes.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.IntakeSystem;
+
+public class IntakeCommandAxes extends Command {
```

Comment:
You can delete this command, we won't be using it anymore. It was just for testing.

Reply:
done